core/auth/interfaces.go

```diff
@@ -237,6 +237,8 @@ type APIs interface {
 	CreateAdminAccount(authenticationType string, appID string, orgID string, identifier string, profile model.Profile, privacy model.Privacy, username string, permissions []string,
 		roleIDs []string, groupIDs []string, scopes []string, creatorPermissions []string, clientVersion *string, l *logs.Log) (*model.Account, map[string]interface{}, error)
 
+	//CreateAdminAccounts creates an accounts for a new admin user
+	CreateAdminAccounts(partialAccount []model.PartialAccount, creatorPermissions []string, clientVersion *string, l *logs.Log) ([]model.Account, map[string]interface{}, error)
```

Comment:
- Add empty lines as the current file is formated.
- The function should be called CreateAccounts